ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -82,6 +82,10 @@ message ServiceDescriptor {
   //   - Key: "ni/service.tags"
   //   - Expected format: serialized JSON string of an array of strings
   //   - Example: "[\"powerup\", \"current\"]"
+  // - Programminglanguage
```

Comment:
Your team needs to update the copy of this proto in the ni-apis repo too.

Reply:
Closing as duplicate of https://github.com/ni/measurementlink-python/pull/599#discussion_r1470121022